tests/core/pm-module/test_ens_integration.py

```diff
@@ -0,0 +1,156 @@
+import pytest
+
+from ens import ENS
+from web3 import Web3
+from web3.exceptions import (
+    BadFunctionCallOutput,
+    PMError,
+)
+
+try:
+    from ethpm import (
+        ASSETS_DIR,
+    )
+    from web3.pm import (
+        PM,
+    )
+except ImportError:
+    ethpm_installed = False
+else:
+    ethpm_installed = True
+
+
+def bytes32(val):
+    if isinstance(val, int):
+        result = Web3.toBytes(val)
+    else:
+        raise TypeError('val %r could not be converted to bytes')
+    if len(result) < 32:
+        return result.rjust(32, b'\0')
+    else:
+        return result
+
+
+@pytest.fixture
+def ens_setup(solc_deployer):
+    ENS_MANIFEST = ASSETS_DIR / 'ens' / '1.0.1.json'
```

Comment:
I started to pester you to move this up to module level but realized why it's here (conditional import failures).  Worth a comment anytime we do something abnormal like this.